packages/backend/src/fee/service.ts

```diff
@@ -45,7 +45,9 @@ async function getFeesPage(
   assetId: string,
   pagination?: Pagination
 ): Promise<Fee[]> {
-  const query = Fee.query(deps.knex).where({ assetId }).getPage(pagination)
+  const query = Fee.query(deps.knex)
+    .where({ assetId })
+    .getPage(pagination, 'desc')
```

Comment:
I think we should rather pass the sorting order as a parameter. That way, we can have a toggle on the Admin UI in the future allowing us to sort both ways.